doc/user/content/sql/system-catalog.md

```diff
@@ -262,56 +262,56 @@ Field             | Type       | Meaning
 `sent`            | [`bigint`] | The number of messages sent.
 `received`        | [`bigint`] | The number of messages received.
 
-### `mz_worker_materialization_dependencies`
+### `mz_worker_compute_dependencies`
 
-The `mz_worker_materialization_dependencies` source describes the dependency structure
+The `mz_worker_compute_dependencies` source describes the dependency structure
 between each [dataflow] and the sources of their data. To create a complete dependency
 structure, the `import_id` column includes other dataflows.
 
 Field      | Type       | Meaning
 -----------|------------|--------
-`export_id`| [`text`]   | The ID of the object that created the dataflow. Corresponds to [`mz_materializations.object_id`](#mz_materializations).
-`import_id`| [`text`]   | The ID of the storage source. Corresponds to [`mz_sources.id`](#mz_sources) or [`mz_tables.id`](#mz_tables) or [`mz_materializations.object_id`](#mz_materializations).
+`export_id`| [`text`]   | The ID of the object that created the dataflow. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).
+`import_id`| [`text`]   | The ID of the storage source. Corresponds to [`mz_sources.id`](#mz_sources) or [`mz_tables.id`](#mz_tables) or [`mz_compute_exports.export_id`](#mz_compute_exports).
```

Comment:
I found a previously present nit: The contents of [`mz_compute_exports.export_id`] include either materialized views or indexes, so in the case of indexes, we are not talking about a storage source. Perhaps we can rephrase the documentation to say something like 'The ID of the source object for the dataflow' instead of referring to a 'storage source'?

This was not part of the PR changes, but now that we fixed the nomenclature, it just jumped at me.